index.html

```diff
@@ -8,6 +8,36 @@
   </head>
   <body class="page">
     <div class="page__container">
+      <section class="advantages"> 
+        <div class="advantages__preview"><p class="advantages__heading">Устали от серости мегаполиса?</p><p class="advantages__heading-text">Нам есть, что вам<br> предложить!</p></div>
+        <div class="advantages__container">
```

Comment:
@NazarPauk I believe that renaming `advantages__container` to `advantages__content` could be really beneficial here. This adjustment would provide a clearer understanding of the section's structure to other developers, indicating that it contains the main content of our section.
```html
<section class="advantages">
  <div class="advantages__intro">...</div>
  <div class="advantages__content">...</div>
</section>
```

Reply:
Done✅